packages/toolpad-app/src/toolpad/AppEditor/AppEditorShell.tsx

```diff
@@ -77,6 +84,7 @@ function CreateReleaseDialog({ appId, open, onClose }: CreateReleaseDialogProps)
                 multiline
                 rows={5}
                 {...register('description')}
+                onKeyDown={handleKeyDown}
```

Comment:
Would it make sense to capture the event higher up in the tree?

Reply:
I guess if we had multiple input fields, maybe, otherwise I'm not sure, would it? 🤔

Reply:
>  guess if we had multiple input fields

We don't have now, but we do have buttons that can be focused. 

> would it?

If the next contributor comes in and adds a checkbox, will the Cmd+Enter work out of the box or will they have to rebuild the feature from this PR?

Reply:
Ah, I think I never tried to submit a form that way while being focused on checkbox/radio, good point, I will make an adjustment 👍

Reply:
And by extension, there are several dialog forms in the application. Is there a way we can make this uniform everywhere, now and for future dialog forms?

Reply:
Sure, I think I can adjust `DialogForm` component for that